lib/msal-common/src/telemetry/performance/PerformanceClient.ts

```diff
@@ -55,6 +56,7 @@ export abstract class PerformanceClient implements IPerformanceClient {
         this.callbacks = new Map();
         this.eventsByCorrelationId = new Map();
         this.measurementsById = new Map();
+        this.httpVer = "1.1";
```

Comment:
I assume this is meant to be a default? Can we assume that if we don't receive the header, that it will be 1.1? And even if that is true now, it will likely not be true in the future.

Reply:
Anything not explicitly set should default to undefined or should not be reported in our telemetry.